code/citadel/cit_virology.dm

```diff
@@ -0,0 +1,244 @@
+
+/datum/symptom/desensitize/Activate(datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		var/mob/living/M = A.affected_mob
+
+		switch(A.stage)
+			if(1)
+				to_chat(M,"<span class='notice'>[pick(uncomfortable_thoughts)]</span>")
+			if(2)
+				M.adjustArousalLoss(-1)
+				to_chat(M,"<span class='notice'>[pick(uncomfortable_thoughts)]</span>")
+			if(3)
+				M.adjustArousalLoss(-2)
+				to_chat(M,"<span class='notice'>[pick(uncomfortable_thoughts)]</span>")
+			if(4)
+				to_chat(M,"<span class='notice'>[pick(discouraged_thoughts)]</span>")
+				M.adjustArousalLoss(-3)
+			else
+				to_chat(M,"<span class='notice'>[pick(discouraged_thoughts)]</span>")
+				M.adjustArousalLoss(-6)
+	return
+
+/datum/symptom/desensitize/Start(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	to_chat(M,"<span class='notice'>You feel strangely uncomfortable.</span>")
+	return
+
+/datum/symptom/desensitize/End(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	to_chat(M,"<span class='notice'>You feel less numb again.</span>")
+	return
+
+
+/*
+//////////////////////////////////////
+Extendong
+
+	Obviously noticable
+	Increases Resistance
+	Slightly faster (quickshot, *laughtrack noises*)
+	Lowers transmission
+	Pretty low Level.
+	Not severe
+Bonus
+	Contrary to popular belief, this makes all genitals increase in size
+	The bane of tiny traps
+//////////////////////////////////////
+*/
+
+/datum/symptom/macro_genitals
+
+	name = "Extendong"
+	stealth = -3
+	resistance = 2
+	stage_speed = 1
+	transmittable = -2
+	level = 2
+	severity = 1
+	var/big_thoughts = list("You feel a bit stiff.","Your loins itch.","You feel like you don't want to wear clothing.")
+
+
+/datum/symptom/macro_genitals/proc/grow_genitalia(mob/living/carbon/human/M)
+	if(M.has_penis())
+		var/obj/item/organ/genital/penis/O = M.getorganslot("penis")
+		if(O.length < COCK_SIZE_MAX) //Couldn't find a macro or anything for the max size of a dong, so hardcoded, sadly.
+			O.length = O.length + 1
+			O.update()
+			to_chat(M,"<span class='warning'>[pick("You feel your cock grow.","Your penis twitches and grows larger.","You feel your dong extend.")]</span>")
+/* REMOVING UNTIL GENITAL FIX PR IS MERGED
+	if(M.has_breasts())
+		var/obj/item/organ/genital/breasts/O = M.getorganslot("breasts")
+		var/size = breasts_size_list.find(O.size)
+		if(size != breasts_size_list(breasts_size_list.len)) //This should be the biggest size of breasts
+			O.size = breasts_size_list(size + 1)
+			O.update()
+			to_chat(M,"<span class='warning'>[pick("Your boobs suddenly gain several cupsizes!","You breasts rapidly expand outward.","You feel your chest grow much larger.")]</span>")
+*/
+	return
+
+/datum/symptom/macro_genitals/Activate(datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		//I think genitalia code is implemented on human level only, so let's not mess up
+		var/mob/living/carbon/human/M = A.affected_mob
+		if(M)
```

Comment:
so there's no point in doing if(M) then.
I guess it makes sense, none of the other viruses seem to do that.